### ISSUE TITLE
Warning Messages

### ISSUE THREAD

**Issue by rovertdude** [TotalFreedom/TotalFreedomMod#1866]
Is there a way that a warning message could be displayed to admins only for the following actions:

Person kills someone in /god or /gmc
Person attempts a command on another user like /ban
Person uses a tag such as SA
Person uses a nickname including &0, &k, &n, or &m
Person uses a nickname such as Hitler or a different troll name
Person teleports another user to them
Person says something like "I'll ddos the server"
Person nicks another user
and anything basic like this.

meant to be w/ #1866 but cant edit on mobile apparently
Perhaps a system could be added to automatically smite a user if no admins are on and someone does the following:
Kill someone in gmc or god
Nick another user
Clear someone elses inventory
and other events that happen when no admins are on.
Example:
happyfyfig7hf6 killed yryjgtigey in creative mode, it would auto smite them for:
[TotalFreedomMod] Creative pvp is not allowed. [Automatic Smite]
or something similar.

**Comment by foxcqrn** [comment]
Good idea @rovertdude. If this isn't accepted, I might consider it for FB3
